traffic_ops/app/lib/API/Server.pm

```diff
@@ -726,41 +746,26 @@ sub get_servers_by_type {
 	my $orderby           = $self->param('orderby') || "hostName";
 	my $orderby_snakecase = lcfirst( decamelize($orderby) );
 
+	my $forbidden;
```

Comment:
This function previously filtered records by the ds_user table (only servers used for the DSes viewable to the user)
We agreed on mail to remove the test and put "is-oper"